omnibus/config/patches/snowflake-connector-python-py2/cryptography-dependency.patch

```diff
@@ -3,7 +3,7 @@ index 0da6a8a..9df4fb5 100644
 --- a/setup.py
 +++ b/setup.py
 @@ -188,8 +188,8 @@ setup(
-         'pytz<2021.0',
+         'pytz<2021.1',
```

Comment:
Shouldn't it be like this?
```suggestion
+         'pytz<2021.1',
```
I think it's safer if you clone the `git` repository and get the diff that `git diff` gives you